tests/providers/test_generic_provider.py

```diff
@@ -54,8 +84,8 @@ def test_parse_pubdate(p):
     actual = sut.parse_pubdate(parsed_date, human_time=ht, timezone=tzone)
 
     # Calculate the difference for human date comparison
-    if ht:
-        actual = (datetime.now(tz.tzlocal()) - actual).seconds
+    if ht and actual:
+        actual = int((datetime.now(tz.tzlocal()) - actual).total_seconds())
```

Comment:
I needed to use total_seconds(). As when using .seconds. It would result in `0` for exactly one day ago / 24 hours ago.